tasks/openshift_packages_playbook.yml

```diff
@@ -5,12 +5,12 @@
 
 - name: Download oc and kubectl
   ansible.builtin.get_url:
-    url: https://mirror.openshift.com/pub/openshift-v4/amd64/clients/ocp/latest/openshift-client-linux-4.13.11.tar.gz
+    url: https://mirror.openshift.com/pub/openshift-v4/amd64/clients/ocp/latest/openshift-client-linux-4.15.0.tar.gz
```

Comment:
please update according to https://github.com/RedHatQE/qe-tools-base-image/blob/main/Dockerfile#L18